project/AppDependencies.scala

```diff
@@ -8,12 +8,12 @@ object AppDependencies {
 
   private val scalaTestVersion = "3.2.12"
   private val scalaTestPlusPlayVersion = "5.1.0"
-  private val httpCachingClientVersion = s"9.5.0$playVersion"
-  private val playConditionalFormMappingVersion = s"1.9.0$playVersion"
+  private val httpCachingClientVersion = s"9.6.0$playVersion"
+  private val playConditionalFormMappingVersion = s"1.11.0$playVersion"
   private val bootstrapVersion = "5.16.0"
```

Comment:
Should this PR also do the bump for bootstrap to `v7.4.0`? Or, is there going to be a separate JIRA ticket for that?